docs/docs/vite-migration.md

```diff
@@ -0,0 +1,95 @@
+## Migration guide for `vite`
+
+This document outlines the steps to migrate your `sku` project to use `vite` as the bundler. The migration process is relatively straightforward and should not require any major changes to your codebase.
+
+### Migrate to `sku/vite/loadable`
+
+> [NOTE!]: If you're not using `sku/@loadable/component` you can skip this step.
+
```

Comment:
I'm not sure if these work on our docs site. Best to just use a regular blockquote.
```suggestion
> If you're not using `sku/@loadable/component` you can skip this step.

```